src/components/Swap/Summary/index.tsx

```diff
@@ -25,18 +25,18 @@ export default Summary
 
 enum ReviewState {
   REVIEWING,
-  ALLOWANCE_PENDING,
+  ALLOWANCE_PROMPTED,
```

Comment:
i'm having a hard time following this name. does this just mean any state before the allowance is fully set? i.e. awaiting permit2 approval, awaiting confirmation of permit2 approval on chain, or awaiting permit signature?

Reply:
I can change it to something else, prompted made sense at one point in my refactor but probably doesn't anymore. But yeah it's basically just any allowance state. I didn't want to call it pending anymore because I felt that could be confused with the approvalLoading variable... which isn't 1-1 with this state.